applications/GeoMechanicsApplication/tests/cpp_tests/custom_utilities/test_node_utilities.cpp

```diff
@@ -78,4 +93,42 @@ KRATOS_TEST_CASE_IN_SUITE(NodeUtilities_UpdatesEverythingWhenNoComponentIsFixed,
     KRATOS_EXPECT_VECTOR_NEAR(expected_vector, node.FastGetSolutionStepValue(ACCELERATION, 0), 1e-12)
 }
 
+KRATOS_TEST_CASE_IN_SUITE(AssignUpdatedVectorVariableToNodes_UpdatesEverythingRegardlessOfFixes,
+                          KratosGeoMechanicsFastSuiteWithoutKernel)
+{
+    // Arrange
+    auto p_node          = make_intrusive<Node>(1, 0.0, 0.0, 0.0);
+    auto nodes_container = NodeContainerWithOneFixedDOF(p_node);
+
+    // Act
+    const array_1d<double, 3> new_values{1.0, 2.0, 4.0};
```

Comment:
`new_acceleration_vector` rather than `new_values`?

```suggestion
    const array_1d<double, 3> new_acceleration_vector{1.0, 2.0, 4.0};
```

Reply:
done